### ISSUE TITLE
smelli 2.0

### ISSUE THREAD

**Issue by DavidMStraub** [smelli/smelli#20]
- [x] Update flavio to 2.0
- [x] Add observables from flavio 2.0
- [x] #19 
- [x] #18 
- ~#12~ (after next flavio version)
- [x] Update API docs
- [x] #23

**Comment by peterstangl** [comment]
@DavidMStraub it seems the only thing that remains before v2.0 can be released is to update the API docs. How do you update them?

**Comment by DavidMStraub** [comment]
```
python3 -m pip install --user pdoc
git checkout git@github.com:smelli/smelli.github.io.git
cd smelli.github.io
make
git add -A
git commit -m "Update API docs"
git push
```
:wink:

**Comment by DavidMStraub** [comment]
(
BTW, for flavio there is no make file but instead a scipt `_flaviomake.py` that builds the static files and `_makedoc.sh` that builds the API docs.

In both cases, the website can be previewed locally with `bundle exec jekyll serve` ([docs](https://help.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll)).
)

**Comment by peterstangl** [comment]
@DavidMStraub after all points in this issue have been addressed, I would publish smelli 2.0 tomorrow. Or is there something still missing?

**Comment by DavidMStraub** [comment]
No, go smelli :tada:
